google/cloud/bigtable/batcher.py

```diff
@@ -24,6 +27,74 @@ class MaxMutationsError(ValueError):
     """The number of mutations for bulk request is too big."""
 
 
+class BatcherIsClosedError(ValueError):
+    """Batcher is already closed and not accepting any mutation."""
+
+
+class _MutationsBatchQueue(object):
+    """Private Threadsafe Queue to hold rows for batching."""
+
+    def __init__(self, max_row_bytes=MAX_ROW_BYTES, flush_count=FLUSH_COUNT):
```

Comment:
as a user, how can i reconfigure max bytes and flush count? (This can come in a later pr)

Reply:
These values gets passed from the `MutationsBatcher` constructor.

For example, they can do:
``` 
batcher = MutationsBatcher(max_row_bytes=5*1024, flush_count=50)
```